src/formatDate.js

```diff
@@ -50,7 +50,57 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newDate = '';
+  let index = 0;
+  const dateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat.includes(toFormat[i])) {
+      index = fromFormat.indexOf(toFormat[i]);
```

Comment:
This means that format doesn't change, better add your value right here and use `continue` to move to the next iteration